### ISSUE TITLE
TPO for admins only

### ISSUE THREAD

**Issue by Scuph** [TotalFreedom/TotalFreedomMod#713]
Honestly, it would lessen our work to smite them and tell them not to tp to us, if a player want another player to tp to them, they can just /tpahere player.

**Comment by looperXD** [comment]
Agreed,  If a player have /tptoggle disabled, They do not want anyone to brother them.

**Comment by PattyToTheWatty** [comment]
Yes this should be implemented

**Comment by SlicedCaek** [comment]
I was going to post this...
Very good idea, because people don't want to be bother'd like when they are building and stuff...

**Comment by jjkatz613** [comment]
Vouch!!

**Comment by AlexFreedomMod** [comment]
This sounds like a good idea, Players cant tpo to them and grief :+1:.

**Comment by PacksGamingHD** [comment]
Vouch

**Comment by Robolawrence** [comment]
PLEASE.

**Comment by Scuph** [comment]
Lol @Robolawrence

**Comment by tylerhyperHD** [comment]
#797 seems like a better idea to me tbh.

**Comment by Wild1145** [comment]
So to my knowledge tpo is now restricted to SA + via the command blocker. The question is are we happy with this implementation. Personally I am happy to say that's a reasonable fix to the problem, opinions people?

**Comment by JeromSar** [comment]
With a configuration file sync, this will indeed be changed. Therefore, this is now related to #2002.
